module/Order/src/Order/Entity/Order.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+namespace Order\Entity;
+
+use DateTime;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity
+ * @ORM\Table(name="orders")
+ */
+
+class Order {
+    /**
+     * @ORM\Id
+     * @ORM\Column(type="integer")
+     * @ORM\GeneratedValue
+     **/
+    protected $id;
+
+    /**
+     * @ORM\Column(type="integer")
+     **/
+    protected $order_no;
+
+    /**
+     * @ORM\Column(type="datetime")
+     **/
+    protected $created;
+
+    /**
+     * @ORM\OneToOne(targetEntity="Customer")
```

Comment:
https://groups.google.com/forum/#!topic/doctrine-user/8Wq-eCou3vg

currently there is absolutly no difference between the two, however on
the OneToOne column there should be a unique constraint, instead of just
an index. This is a known issue which will be adressed in the next
released. Other than the Unique constraint there is no difference
between the two.